projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,120 @@
+
+* As the Developer Lead, Ockert van Schalkwyk is responsible for maintaining an Enterprise level standard for the project’s software code.  He will review and approve changes, forks, and merges.  His background as senior Mendix developer with a focus on systems integrations, makes him the ideal candidate for the role.    
+
+* As a Mendix Solution architect, Stephan Bruijnis is responsible for guiding architectural integrity of the project.  He will ensure the project upholds and maintains targeted privacy, security, and interoperability standards. His extensive experience as a Senior Mendix architect makes him ideal for this project.    
+
+* As the Methodology and Mechanics Lead, Ned Gosaynie is responsible for maintaining documentation and versioning for the project.  He will review and approve the technical documentation needed to support the project needs. Additionally, he will manage versioning of the software. His experience as the of Head of Platform Services makes him the ideal candidate for this position.   
+
+ 
+The team’s philosophy for the Farmworker Wallet OS project will be a continuation of the social good principles that have driven the project since its origin. Having worked together for years, the team has a well-established working relationship. Decisions will be evaluated through the lens of farmworker benefit, security, privacy, and impact.   
+
+# Links to Documented Governance Practices
+_Include links to any documented governance practices._
+
+# Preferred Maturity Level
+Labs
+
+# Communication Channels
+Slack
+
+# Project Website
+TBD
+
+# Social Media
+* [linkedin.com/company/Entidad](https://www.linkedin.com/company/entidad/)
+* [medium.com/@Entidad](https://medium.com/@Entidad)
+
+# Financial Sponsorship
+UFW Foundation
+
+# Infrastructure
+The **Farmworker Wallet OS** will be organized and published as a suite of software modules that can be imported into any existing Mendix app code repository. The Mendix software modules effectively serve as a digital wallet SDK that can be embedded into any Mendix application to enhance the user experience. The code repository for this project is itself a Mendix app repository and as such can be executed locally on any [supported](https://docs.mendix.com/refguide9/system-requirements/) developer workstation. 
+
+
+The initial version of the digital wallet SDK is being built on top of [Aries Framework Javascript](https://github.com/hyperledger/aries-framework-javascript), an open-source framework maintained by the [Hyperledger Aries developer community](https://www.hyperledger.org/use/aries) helping to foster participation with the Aries Digital Trust ecosystem. Over time, the digital wallet SDK can be extended to support other digital trust open-source frameworks such as OPENID4VC and VC API. 
```

Comment:
Aries Framework Javascript is a dependency, correct? There is no plan to fork this code base, is there?

Reply:
Correct, I've updated the project proposal with the AFJ v0.4.0 dependency. We have no plans to fork the AFJ codebase and plan to contribute to it where we can.